tithe.lic

```diff
@@ -0,0 +1,66 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#tithe
+=end
+
+custom_require.call(%w[common common-money common-travel events drinfomon])
+
+class TitheActions
+  include DRC
+  include DRCT
+  include DRCM
+
+  def initialize
+    arg_definitions = [[]]
+
+    args = parse_args(arg_definitions, true)
+
+    @settings = get_settings(args.flex)
+    @hometown = @settings.hometown
+    @pray_to_chadatru = @settings.pray_to_chadatru
```

Comment:
maybe put these 3 chadatru lines in an if DRStats.paladin? conditional block so clerics dont get a useless uservar